src/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../.eslintrc",
+  "env": {
+    "browser": true
+  },
+  "rules": {
+    "strict": [2, "global"]
```

Comment:
The best part about `eslint` for me:

We can lint the whole project with no config repetition. `eslint` is smart enough to extend local and root config options.